bundles/org.openhab.ui/web/src/components/widgets/system/oh-input.vue

```diff
@@ -21,37 +17,65 @@
 </style>
 
 <script>
+import dayjs from 'dayjs'
+
 import mixin from '../widget-mixin'
 import { OhInputDefinition } from '@/assets/definitions/widgets/system'
 
 export default {
   mixins: [mixin],
   widget: OhInputDefinition,
-  mounted () {
-    delete this.config.value
+  data () {
+    return {
+      pendingUpdate: null
+    }
   },
   computed: {
     value () {
       if (this.config.variable && this.context.vars[this.config.variable] !== undefined) {
         return this.context.vars[this.config.variable]
-      } else if (this.config.showOK && this.context.vars[this.config.value] !== undefined) {
-        return this.context.vars[this.config.value]
+      } else if (this.config.sendButton && this.pendingUpdate !== null) {
+        return this.pendingUpdate
       }
       return (this.config.item) ? this.context.store[this.config.item].state : this.config.defaultValue
```

Comment:
I had it changed to something like this (which fit my use case):
```
      } else if (this.config.item && (this.context.store[this.config.item].state !== "NULL") && (this.context.store[this.config.item].state !== "UNDEF")) {
        return this.context.store[this.config.item].state
      }
      return this.config.defaultValue
```

Reply:
Yes, makes sense, thanks!